src/plexil-adapter/OwInterface.cpp

```diff
@@ -517,6 +604,10 @@ OwInterface::~OwInterface ()
   if (m_socSubscriber) delete m_socSubscriber;
   if (m_rulSubscriber) delete m_rulSubscriber;
   if (m_guardedMoveSubscriber) delete m_guardedMoveSubscriber;
+  // if (m_systemFaultMessagesSubscriber) delete m_systemFaultMessagesSubscriber;
+  // if (m_armFaultMessagesSubscriber) delete m_armFaultMessagesSubscriber;
+  // if (m_powerFaultMessagesSubscriber) delete m_powerFaultMessagesSubscriber;
+  // if (m_ptFaultMessagesSubscriber) delete m_ptFaultMessagesSubscriber;
```

Comment:
Why are you still holding on them? :roll_eyes:
```suggestion
```

Reply:
lol forgot to delete!